deployments/helm-chart/crds/appprotectdos.f5.com_dosprotectedresources.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This file is the same as the one above, is there any way to avoid the duplication?

Reply:
looks like all the other crds are in here. If there is a change to make in this regard, this PR is not the place for it.